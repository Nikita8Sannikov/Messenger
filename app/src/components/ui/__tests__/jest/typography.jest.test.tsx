import { screen, render } from "@testing-library/react";
import TypographyComponent from "@components/ui/typography";

describe("Typography", () => {
    test("Typography with children", () => {
        render(<TypographyComponent>Test</TypographyComponent>)
        const typographyElement = screen.getByText("Test")
        expect(typographyElement).toBeInTheDocument()
    })

    test("matches snapshot", () => {
        const { asFragment } = render(<TypographyComponent>Test</TypographyComponent>);

        // Снимаем снапшот компонента
        expect(asFragment()).toMatchSnapshot();
    });
})
