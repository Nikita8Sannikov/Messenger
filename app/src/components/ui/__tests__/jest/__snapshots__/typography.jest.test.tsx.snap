// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography matches snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
  >
    Test
  </p>
</DocumentFragment>
`;
